descriptors/annotators/io/LoadPointCloud.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<taeDescription xmlns="http://uima.apache.org/resourceSpecifier">
+  <frameworkImplementation>org.apache.uima.cpp</frameworkImplementation>
+  <primitive>true</primitive>
+  <annotatorImplementationName>rs_loadPointCloud</annotatorImplementationName>
+  <analysisEngineMetaData>
+    <name>LoadPointCloud</name>
+    <description>Load Point clouds from files.</description>
+    <version>1.0</version>
+    <vendor/>
+    <configurationParameters>
+        <configurationParameter>
+            <name>folder</name>
+            <type>String</type>
+            <multiValued>false</multiValued>
+            <mandatory>true</mandatory>
+        </configurationParameter>
+        <configurationParameter>
+            <name>filenames</name>
+            <type>String</type>
+            <multiValued>true</multiValued>
+            <mandatory>true</mandatory>
+        </configurationParameter>
+    </configurationParameters>
+    <configurationParameterSettings>
+        <nameValuePair>
+            <name>folder</name>
+            <value>
+                <string>/home/anindex/</string>
+            </value>
+        </nameValuePair>
+        <nameValuePair>
+            <name>filenames</name>
+            <value>
+              <array>
```

Comment:
Yeah, my intention is to let the user specify their clouds in order. However, it can be tedious to pass a list like that. As you said, to specify the order to load point cloud, user should follow naming convention. I will add the option to loop or not and let the AE find all specific pcd file.